### PR TITLE
HL-154

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-154)

- Extracts out nested ternary

### CODE REVIEW

File: src/pages/Bets/components/BetRows.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { Row } from "./BetTable";
+
+type Props = {
+  bets?: BetHistory[];
+  onClickBet: (bet?: BetHistory) => void;
+};
+
+const BetRows: React.FC<Props> = ({ bets, onClickBet }) => (
+  <React.Fragment>
+    {!bets ? (
```
Comment (xiangzhe0812) [inline src/pages/Bets/components/BetRows.tsx:12]: should be bets.length===0?

File: src/pages/Bets/components/BetRows.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      <td className="p-2 select-none">loading...</td>
+    ) : (
+      bets.map(v => (
```
Comment (xiangzhe0812) [inline src/pages/Bets/components/BetRows.tsx:15]:
```suggestion
      bets.map(bet => (
```

File: src/pages/Bets/components/BetRows.tsx (hunk 1 above)
Comment (brettvniekerk) [reply src/pages/Bets/components/BetRows.tsx:12]: `bets` is undefined while the data loads from the api

File: src/pages/Bets/components/BetRows.tsx (hunk 2 above)
Comment (brettvniekerk) [reply src/pages/Bets/components/BetRows.tsx:15]: 👍

**Review by xiangzhe0812 — APPROVED** [review]

**Review by OptimusOpus — APPROVED** [review]

**Pull request merged**